cli/lib/tasks/unzip.js

```
@@ -113,13 +157,20 @@ const unzip = ({ zipFilePath, installDir, progress }) => {
           case 'darwin':
             return unzipWithOsx()
           case 'linux':
+            return unzipWithUnzipTool()
```

but will every Linux system include `unzip`?

---

no, but it silently falls back to node.js unzip if it fails